docs/examples/others/large_full_volume_seg_gradcam.md

```
@@ -0,0 +1,294 @@
+        mask_type="all",
+        show_progress=True,
+    ),
+    info=info,
+    num_classes=1,
+)
+
+full_cam = normalize_full_cam(full_cam)
+print("input shape:", x.shape)
+print("full_cam shape:", full_cam.shape)
+```
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)